src/dataprotection/azext_dataprotection/manual/aaz_operations/backup_instance.py

```diff
@@ -0,0 +1,254 @@
+        args_schema.source_data_store_type._required = False
+        args_schema.source_data_store_type._registered = False
+
+        args_schema.restore_request_object = AAZFreeFormDictArg(
+            options=["--restore-request-object"],
+            help="Gets or sets the restore request object. Expected value: json-string/@json-file.",
+            required=True,
+        )
+
+        return args_schema
+
+    class BackupInstancesTriggerRestore(_RestoreTrigger.BackupInstancesTriggerRestore):
+
+        @property
+        def content(self):
+            return convert_dict_keys_snake_to_camel(self.ctx.args.restore_request_object.to_serialized_data())
+
+        def on_200(self, session):
+            data = self.deserialize_http_content(session)
+            data = data.get('properties')
+            self.ctx.set_var(
+                "instance",
+                data,
+                schema_builder=self._build_schema_on_200
+            )
+
+
+class ValidateForBackup(_ValidateForBackup):
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super(_ValidateForBackup, cls)._build_arguments_schema(*args, **kwargs)  # pylint: disable=bad-super-call
+
+        _args_schema = cls._args_schema
+        _args_schema.resource_group = AAZResourceGroupNameArg(
+            required=True,
+        )
+        _args_schema.vault_name = AAZStrArg(
+            options=["--vault-name"],
+            help="The name of the backup vault.",
+            required=True,
+            id_part="name",
+        )
+        _args_schema.backup_instance = AAZFreeFormDictArg(
+            options=["--backup-instance"],
+            help="Backup Instance",
+            required=True,
+        )
+        print("args schema")
+        return cls._args_schema
+
+    class BackupInstancesValidateForBackup(_ValidateForBackup.BackupInstancesValidateForBackup):
+
+        # TODO: Reach out to swagger team about potentially fixing this in the swagger-side.
+        #       We have to replace "final-state-via" to "location" instead of "azure-async-operation"
+        # In case of debug issues, compare against the equivalent call in
+        #  src\dataprotection\azext_dataprotection\aaz\latest\dataprotection\backup_instance\_validate_for_backup.py
+        #  to see if there are any new divergences.
+        def __call__(self, *args, **kwargs):
+            request = self.make_request()
+            session = self.client.send_request(request=request, stream=False, **kwargs)
+            if session.http_response.status_code in [202]:
+                return self.client.build_lro_polling(
+                    self.ctx.args.no_wait,
+                    session,
+                    self.on_200,
+                    self.on_error,
+                    lro_options={"final-state-via": "location"},
```

Comment:
You should fix the swagger

Reply:
Planned the fix, but it will take some time as discussed offline. This exact function is already committed to the repo (it is showing up again due to code cleanup during the migration effort): https://github.com/Azure/azure-cli-extensions/blob/0b2cbaab7ecd11a8cd99eb5d455045abb22ee9c9/src/dataprotection/azext_dataprotection/manual/custom.py#L180.